src/main/java/com/coveo/pushapiclient/DocumentUploadQueue.java

```diff
@@ -8,11 +8,11 @@
 /** Represents a queue for uploading documents using a specified upload strategy */
 class DocumentUploadQueue {
   private static final Logger logger = LogManager.getLogger(DocumentUploadQueue.class);
-  private final UploadStrategy uploader;
-  private final int maxQueueSize = 5 * 1024 * 1024;
-  private ArrayList<DocumentBuilder> documentToAddList;
-  private ArrayList<DeleteDocument> documentToDeleteList;
-  private int size;
```

Comment:
If the outcome of the comment below on extending this class is it's not worthwhile, maybe these variables could remain `private`

Reply:
As per my previous comment, we are not overriding all methods in the parent class.